manifests/ha/base/overlays/argocd-application-controller-statefulset.yaml

```diff
@@ -14,4 +14,4 @@ spec:
         - --operation-processors
         - "10"
         - --redis
-        - "argocd-redis-ha-haproxy:6379"
+        - "argocd-redis-ha-haproxy:6379"
```

Comment:
Can you add the missing new-line at eof?

Reply:
Done.